src/test/java/schools/SchoolTest.java

```diff
@@ -27,76 +25,110 @@ public void clearDataAfterTest(){
         school.getSchoolClasses().clear();
     }
 
+    /**
+     * uwaga ogolna - za mala probla danych testowych i ich przypadkow. Czy na pewno dobrze zadzialaja metody liczace ucznikow jak bedzie ich wiecej niz 1 w klasie
+     * i wiecej klas? Na razie nie ma takiej pewnosci.
+     * A co te klasy maja zwrocic jak nie ma zadnej klasy?
+     * Czy w ogole mozna utworzyc szkole bez klasy?
+     * Czy mozna dodac klase bez zadnego ucznia?
+     *
+     * Jakiekolwiek sa odpowiedzi na ww pytania powinny byc dla nich testy.
+     */
```

Comment:
w tej klasie testowała zastosowanie @BeforeEach @AfterEach aby nie musieć tworzyć danych testowych do każdego testu (teraz już wiem że jest to mało wygodne); operację dodania ucznia i stworzenia klasy wykonałam w sekcji @BeforEach, ale sprawdze z większą ilością danych

Reply:
nie bardzo lubie beforeeach poniewaz trzeba skakac miedzy kodem a zainicjowaniem danych. To co zrobilas tutaj mi pasuje i dlatego nie komentowalam. Jak to zostanie, to dodatkowo w niektorych przypadkach warto dodac kolejne dane juz tylko na poziomie niektorych testow